exercises/spiral-matrix/spiral_matrix_test.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from spiral_matrix import spiral
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```

Comment:
Can you move this line down by one to be more consistent with the other test files?

Reply:
Thanks, @N-Parsons ,will do.

Reply:
I believe for full consistency, you the spacing should be 2 blank lines above, one below (see below):

```Python
from spiral_matrix...


# Tests adapted from...

class...